docs/understanding-radicle/faq.md

```diff
@@ -1,14 +1,135 @@
+
+## When CLI tooling?
+We're working on it! We will introduce CLI tooling alongside of Upstream
+development.
+
+## Can I backup a GitHub project on Radicle?
+Yes! Publishing a codebase to Radicle is a great way to create a peer-to-peer
+backup of your repositories. Maintaining a mirror of a project on Radicle is as
+simple as pushing to another remote. Read more about [creating and sharing
+projects][cp].
+
+## Can I replace GitHub with Radicle?
+If you want! While our Beta release will have the basic collaboration features
+(i.e. code hosting, sharing, checking out, and pushing/pulling), we plan to
+introduce features that could support a similar day-to-day code collaboration
+experience to GitHub. They will include bug reporting, patches, code review, and
+discussions. 
+
+That being said, while we believe that reducing one's reliance on
+centrally-hosted platforms is generally a good idea, we also believe that code
+collaboration solutions serve different purposes for different people. Radicle
+Upstream *will* support social collaboration, but it's priority will be
+delivering secure, local-first, peer-to-peer code collaboration — not an exact
+GitHub replica. 
+
+## Where is my data stored? 
+On the Radicle network, content is distributed peer-to-peer via a process called
+gossip. This means that peers self-host their own content — and the content of
+any peers they are interested in — locally on their machine. It also means that
+whenever your data is published to the network, it can be replicated and stored
+on another peer's machine.
+
+## Can I create private repositories on Radicle?
+No, not yet - but in the future! Private projects with end-to-end encryption are
+on our roadmap.
+
+## What is a remote?
+A 'remote' refers to a version of your project that is maintained by another
+person. To collaborate with others on Radicle, you have to add and follow other
+people's remotes to be able to fetch changes from them. You can manage remotes
+on your project page (See [Adding Remotes][ar]). For more on how remote
+repositories work, see the [Git documentation][mr]]. 
+
+## What's a Radicle ID?
+A Radicle ID is a unique way to identify people and projects in the Radicle
+Network. You use a project's Radicle ID to find it on Radicle Upstream. You can
+create a Radicle ID through Radicle Upstream.
+
+## What's a Device ID?
+A Device ID is the encoding of a peer's public key tied to a specific device.
+People will be able to manage multiple Device IDs in the future, but for now can
+only have one Device ID per identity.
+
+## How does Radicle interact with Git?
+Radicle basically uses Git as a database. This means everything is stored in a
+single Git monorepo that is read and written from via the Upstream client. To
+read more about Radicle's Git Implementation, see [How it Works][hw].
+
+## What does `following` mean in Radicle?
+Following (also known as [Tracking][tr]) a project replicates its data to your
```

Comment:
Is it also known as tracking, seems strange to have two names for it.

Reply:
protocol level is tracking. I can change language to make it less strange

Reply:
Yeah I think we should be explicit -- if this document is about the protocol, then it should be called "tracking".

Reply:
It's not protocol specific. This is mainly where Upstream users will be directed. I will keep it as Following.